programs/openbook-v2/tests/program_test/client.rs

```diff
@@ -249,11 +237,12 @@ pub struct CreateMarketInstruction {
     pub quote_mint: Pubkey,
     pub base_vault: Pubkey,
     pub quote_vault: Pubkey,
-    pub market_index: MarketIndex,
     pub name: String,
     pub bids: Pubkey,
     pub asks: Pubkey,
     pub event_queue: Pubkey,
+    pub market: TestKeypair,
+    pub market_authority: Pubkey,
```

Comment:
why is this exposed in all the instructions? you can use `find_program_address` given that market address is known. Also, for the rest of the ixs, you can read the field from the market account as it is done for the other market-related accounts (e.g. oracles & event_queue)